airbyte-integrations/connectors/source-google-drive/source_google_drive/source.py

```diff
@@ -15,16 +15,12 @@
 
 
 class SourceGoogleDrive(FileBasedSource):
-    def __init__(self, catalog: Optional[ConfiguredAirbyteCatalog], config: Optional[Mapping[str, Any]], state: Optional[TState]):
-        super().__init__(
```

Comment:
No need to override constructor if we aren't having to pass in classes/objects via constructor.